content/6_Advanced/LineContainer.mdx

```diff
@@ -30,7 +30,195 @@ frequency: 1
 	</Resource>
 </Resources>
 
-<IncompleteSection>Example problem + implementation?</IncompleteSection>
+A half-plane can be defined as a planar region that consists of all points on one side of an infinite straight line, and none on the other side. We can apply computational geometry here, as an intersection can be represented as a convex polygon, where every point inside it is also inside all half-planes. If we can construct this polygon, we can retrieve the intersections.
+
+### Example Problem
+
```

Comment:
if there's only 1 example problem, no need for this header
also the rest of the module doesn't follow the format i gave

Reply:
I removed the header since there's just one focus problem, and I also have the explanation and implementation sections, as detailed in the document. If you have any more suggestions, please specify them (bcz I think the module looks good now).